### PR TITLE
Database confi

### PR DESCRIPTION
**Pull request by Eljuan25**
configure the database from sql to postgresql so that it can be used in that way and in a more efficient way.

### CODE REVIEW

**Review by mtopete1996 — CHANGES_REQUESTED** [review]
Looks good but I'm gonna need you to squash the commits into 1 and edit the message to "Add PostgreSQL configuration and pg gem"

File: config/database.yml [hunk 1]
```diff
@@ -5,21 +5,25 @@
 #   gem "sqlite3"
 #
 default: &default
-  adapter: sqlite3
+  adapter: postgresql
+  encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
-  timeout: 5000
+  username: mind
```
Comment (jesusMava) [inline config/database.yml:11]: No estoy seguro si esto vaya a ocasionar problema para los demas usuarios, usualmente no se pone esta parte

File: config/database.yml [hunk 2]
```diff
@@ -5,21 +5,25 @@
...
+  host: localhost
+  port: 5432
+  extension: postgis
```
Comment (jesusMava) [inline config/database.yml:14]: Pregunta, no necesitas usar la gema 'activerecord-postgis-adapter' para habilitar postgis

Comment (Eljuan25) [reply config/database.yml:14]: maybe

**Review by mtopete1996 — APPROVED** [review]
Looks Good To Me :)

**Review by jesusMava — APPROVED** [review]

**Pull request merged**